CHANGELOG.md

```diff
@@ -4,15 +4,72 @@ All notable changes to this project will be documented (for humans) in this file
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [0.6.0] - 2019-12-05
+
+This is largely a bug fix.  Metadata and order of `_key-value` fields in BIDS
+could change from the result of converting using previous versions, thus minor
+version boost.
+14 people contributed to this release -- thanks
+[everyone](https://github.com/nipy/heudiconv/graphs/contributors)!
+
+### Enhancement
+
+- Use [etelemetry](https://pypi.org/project/etelemetry) to inform about most
+  recent available version of heudiconv. Please set `NO_ET` environment variable
+  if you want to disable it ([#369][])
+- BIDS:
+  - `--bids` flag became an option. It can (optionally) accept `notop` value
```

Comment:
`--bids` was already an option - perhaps
```suggestion
  - `--bids` flag now supports additional options. It can accept `notop` value
```

Reply:
I agree to disagree:
```shell
$> git diff v0.5.4.. -- heudiconv/cli/run.py | grep -2 -e -bids
                         help='session for longitudinal study_sessions, default '
                         'is none')
-    parser.add_argument('-b', '--bids', action='store_true',
-                        help='flag for output into BIDS structure')
+    parser.add_argument('-b', '--bids', nargs='*',
+                        metavar=('BIDSOPTION1', 'BIDSOPTION2'),
+                        choices=['notop'],
+                        dest='bids_options',
+                        help='flag for output into BIDS structure. Can also '
+                        'take bids specific options, e.g., --bids notop.'
+                        'The only currently supported options is'
+                        '"notop", which skips creation of top-level bids '
```

Reply:
so it went from a pure flag (`store_true`) to an option which could be used as a flag as well

Reply:
yes. though i would refer to all flags, even ones that just store_true, as "options". small thing but the way its worded in the changelog confused me

Reply:
hm, indeed it could be said that every flag (a boolean switch) is an option, but not every option is a flag.  I would not say that a flag can supports options (as your wording suggests).  Not sure what better wording would be so proceeding forward.